tracepoints/operational/runtest.sh

```diff
@@ -182,6 +182,32 @@ if [ $? -eq 0 ]; then
     exit 0
 fi
 
+# Warn if gcc does not have retpoline (x86_64) or expoline (s390x) support.
+# SystemTap cannot find any tracepoints in newer kernels with Spectre v2
+# mitigations without retpoline/expoline support in gcc.
+# aarch64 and ppc64le mitigated Spectre v2 through other means so we do
+# not need to check those platforms for gcc support.
+if [ -e /sys/devices/system/cpu/vulnerabilities/spectre_v2 ]; then
+  CFLAGS=""
+  if [ "`uname -i`" = "x86_64" ]; then
+    CFLAGS="-mindirect-branch=thunk-extern"
+    CFLAGS+=" -mindirect-branch-register"
+    MITIGATION="retpoline"
+  elif [ "`uname -i`" = "s390x" ]; then
+    CFLAGS="-mindirect-branch=thunk-extern"
+    CFLAGS+=" -mindirect-branch-table "
+    CFLAGS+=" -mfunction-return=thunk-extern "
```

Comment:
Thanks @jbastian ! Is the space needed at the end here ? Also do you have a test job ?

Reply:
Oops, I see I put both leading and trailing spaces. It doesn't hurt, but it does look ugly. I'll clean that up.

Test job:
https://beaker.engineering.redhat.com/jobs/3966563

Reply:
The patch is updated with the trailing whitespace removed from those two CFLAGS lines.

Reply:
@jbastian it was total nit-pick, looks good! Thanks for the test jobs to see it in action :-)